frontend/src/LineGraph.res

```diff
@@ -290,7 +291,7 @@ let make = React.memo((
     <Row alignX=#right spacing=Sx.md sx=[Sx.w.auto]>
       <Text sx=[Sx.leadingNone, Sx.text.xl2, Sx.text.bold, Sx.text.color(Sx.gray900)]>
         {lastValue->Belt.Option.mapWithDefault("", value =>
-          Js.Float.toFixedWithPrecision(~digits=2, value)
+          Js.Float.toPrecisionWithPrecision(~digits=4, value)
```

Comment:
Any reason why we use `4` digits here and not `6` as in the overview table? Probably don't make sense to be as precise as in the table but then the numbers might not be exactly the same:

![image](https://user-images.githubusercontent.com/5595092/113099863-ebde5480-91fa-11eb-8702-ef8352c4d6b7.png)

I'm not sure if it's really important though. What do you think?

Reply:
The only reason is that in some cases the number gets very long in the graph starting to collide with the title on the left.

I think 4 is probably ok for graph summary values. I started with 4 for tables too, but there we actually have room for 6.